cla_frontend/assets-src/javascripts/app/constants.json

```diff
@@ -1 +1 @@
ance"}, {"text": "Guarantee State Pension Credit", "value": "pension_credit"}, {"text": "Income-related Employment and Support Allowance", "value": "employment_support"}], "RESEARCH_CONTACT_VIA": [{"text": "Email", "value": "EMAIL"}, {"text": "Phone", "value": "PHONE"}, {"text": "Sms", "value": "SMS"}], "DIAGNOSIS_SCOPE": {"INELIGIBLE": "INELIGIBLE", "UNKNOWN": "UNKNOWN", "MEDIATION": "MEDIATION", "OUTOFSCOPE": "OUTOFSCOPE", "INSCOPE": "INSCOPE", "CONTACT": "CONTACT"}, "DISREGARDS": [{"text": "Infected Blood Support Scheme", "value": "infected_blood"}, {"text": "Vaccine Damage Payment", "value": "vaccine_damage"}, {"text": "vCJD Trust", "value": "vcjd_trust"}, {"text": "Criminal Injuries Compensation Scheme", "value": "criminal_injuries"}, {"text": "National Emergencies Trust", "value": "national_emergencies"}, {"text": "London Emergencies Trust", "value": "london_emergencies"}, {"text": "We Love Manchester Emergency Fund", "value": "love_manchester"}, {"text": "Energy prices Act 2022", "value": "energy_prices"}, {"text": "Cost of living payment- Social Security (additional payments) Act 2022", "value": "cost_living"}], "EXEMPT_USER_REASON": [{"text": "Client is a child", "value": "ECHI"}, {"text": "Client is in detention", "value": "EDET"}, {"text": "12 month exemption", "value": "EPRE"}], "REQUIRES_ACTION_BY": {"OPERATOR": "operator", "OPERATOR_MANAGER": "operator_manager", "PROVIDER_REVIEW": "1_provider_review", "PROVIDER": "2_provider"}, "ELIGIBILITY_STATES": {"UNKNOWN": "unknown", "YES": "yes", "NO": "no"}, "EXPRESSIONS_OF_DISSATISFACTION": [{"text": "Believes operator has given incorrect information", "value": "incorrect"}, {"text": "Unhappy with Operator Service determination (Scope)", "value": "scope"}, {"text": "Unhappy with Operator Service determination (Means)", "value": "means"}, {"text": "Wants personal details deleted", "value": "delete"}, {"text": "No response from specialist advisor, or response delayed", "value": "advisor_response"}, {"text": "Operator service - delay in advice", "value": "operator_delay"}, {"text": "Unhappy with operator's attitude", "value": "operator_attitude"}, {"text": "Unhappy with specialist's attitude", "value": "advisor_attitude"}, {"text": "Alternative help not appropriate or not available", "value": "alt_help"}, {"text": "Unhappy with online service", "value": "public_tool"}, {"text": "Problems with adaptations or adjustments", "value": "adaptations"}, {"text": "Scope reassessment requested", "value": "scope_assessment"}, {"text": "Financial reassessment requested", "value": "means_assessment"}, {"text": "Threatens to pass the matter on to the media, or other public or regulatory body", "value": "pass_to_public"}, {"text": "Breach of Data Protection Act/policy and confidentiality", "value": "data_protection"}, {"text": "Discrimination from an operator or specialist", "value": "discrimination"}, {"text": "Client unhappy with PLO referral", "value": "plo_referral"}, {"text": "Other", "value": "other"}]}
```

Comment:
@jogilder @psweeting1 Is it possible to format so it makes diff'ing it easier? Is it not minified by gulp if so can we have this source file in a nicer json format?

Reply:
@said-moj this is not minified by gulp so would need an extra ticket to do this work. I think out of scope.